server/app/services/mongodb/migrator.rb

```diff
@@ -8,7 +8,7 @@ class Migrator
     include DistributedLocks
 
     LOCK_NAME = 'mongodb_migrate'.freeze
-    LOCK_TIMEOUT = 60
+    LOCK_TIMEOUT = (60 * 5)
```

Comment:
Hmm... this timeout is also used for the `with_dlock(LOCK_NAME, LOCK_TIMEOUT)`. Does it make sense to use the same timeout for both the lock aquire and the cleanup?

BTW... if the `Migrator#migrate` -> `with_dlock` times out... then it just returns false, and continues booting up the server without running the migrations :scream: